### PR TITLE
fix: utopia path params

### PR DESCRIPTION
**Pull request by aaryanpunia**

### CODE REVIEW

File: actix-server/src/lib.rs [hunk 1]
```diff
@@ -182,6 +182,10 @@ impl Modify for SecurityAddon {
             models::Org,
             models::Note,
             models::Task,
+            models::Deal,
+            models::Link,
+            models::Email,
+            models::Phone,
```
Comment (drew-harris) [inline actix-server/src/lib.rs:188]: Missing models::Contact

**Review by drew-harris — CHANGES_REQUESTED** [review]
looks good other than this

File: actix-server/src/lib.rs (hunk 1 above)
Comment (aaryanpunia) [reply actix-server/src/lib.rs:188]: lmao I spent so long trying to make sure everything as covered, good catch!

**Review by drew-harris — APPROVED** [review]
LGTM, still gonna defer @cdxker cuz I haven't been granted explicit review/merge permission yet lol

**Pull request merged**